.github/workflows/velox_weekly.yml

```diff
@@ -99,3 +100,68 @@ jobs:
           sudo apt-get install -y openjdk-8-jdk
           export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
           cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+
+  build-on-openeuler:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          echo "JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk" >> $GITHUB_ENV
+      - name: Build
+        run: |
+          echo "JAVA_HOME: $JAVA_HOME"
+          cd $GITHUB_WORKSPACE/
+          ./dev/package.sh
+          cp package/target/thirdparty-lib/gluten-thirdparty-lib-*.jar /root/
```

Comment:
Maybe, we can directly copy the jar to package/target/lib/ if we only use `./deve/package.sh  --spark_version=3.5` to build gluten.